### DISCUSSION TITLE
C/C++ v1.7.1 go to definition always jumps to declearation is really bad

### DISCUSSION THREAD

**Discussion by sshsu** [microsoft/vscode-cpptools#8360] [Q&A]
I find C/C++ v1.7.1 go to definition is really bad,  compare old version, it always jumps to the declearation and slower now, it is baddly inconvinient now, what happen?

**Comment by sean-mcmanus** [comment]
Not sure -- we're not aware of any regression. Are you able to provide a repro? Do you see any crash logging in the C/C++ output panel?
